templates/legal/terms-and-policies/cube-terms.html

```diff
@@ -65,63 +65,80 @@ <h2>2. Account</h2>
+        You may cancel your CUBE account at any time through your account page at <a href="https://cube.ubuntu.com/account/settings">https://cube.ubuntu.com/account/settings</a> by clicking <b>Delete My Account</b>. Note that this does not delete your Ubuntu SSO account, since you might be using it for other Canonical services, neither does it automatically delete your edX account. To delete your Ubuntu SSO account, visit <a href="https://login.ubuntu.com/">https://login.ubuntu.com</a> and click <b>Permanently delete account</b>.
     </p>
     <h2>3. Termination</h2>
     <p>
         We may cease to offer the services for any reason, in which case we will provide you with notice via the email associated with your account.
     </p>
     <h2>4. Changes</h2>
     <p>
-        We continually aim to improve the delivery and content of the Service and accordingly may make changes to the Service from time to time. 
+        We continually aim to improve the delivery and content of the Service and accordingly may make changes to the Service from time to time.
     </p>
     <p>
         New features may be added, but we may also modify or discontinue (temporarily or permanently) part of the Service, in part or in whole. We may also offer Services on a trial/beta basis, in which case your use of those Services will be limited to the period of your trial/beta and subject to the terms of the trial/beta.
     </p>
     <p>
-        In the event of a material change to the Service, we will notify you via the email associated with your account. What constitutes a material change in this circumstance will be determined by Canonical, in good faith and using common sense and reasonable judgment. 	
+        In the event of a material change to the Service, we will notify you via the email associated with your account. What constitutes a material change in this circumstance will be determined by Canonical, in good faith and using common sense and reasonable judgment.
     </p>
     <p>
         Similarly, we may occasionally make changes to these Terms of Service and will notify you of material changes via the email associated with your account. If Canonical does make changes to these Terms of Service, all changes will go into effect at the time we post the updated Terms of Service or as otherwise notified at that time.
     </p>
     <h2>5. Intellectual property and content</h2>
-    <h3>Intellectual property</h3>
+    <h3><u>Intellectual property</u></h3>
```

Comment:
I know these are underlined in the copy doc but checked with @anasereijo and we don't normally underline titles so it's fine to just leave them as a h3